flagging_site/templates/index.html

```diff
@@ -26,5 +26,8 @@ <h2>Current Water Quality</h2>
         {% endif %}
     </div>
     {% endfor %}
+    <div style="position: relative;overflow: hidden;width: 100%;padding-top: 75%">
+        <iframe src=" http://127.0.0.1:5000/flags" style="position: absolute;top: 0;left: 0;bottom: 0;right: 0; width: 100%; height: 100%"></iframe>
```

Comment:
I would not add the iframe to the home page.

That said, we should have this in our back pocket. Maybe as a page in the docs called like "Exporting Data" in a subsection called "HTML iFrame." I would 

```
## HTML iFrame

The outputs of the model can be exported as an iFrame, which allows the website's live data to be viewed on a statically rendered web page (such as those hosted by Weebly). To export the data using an iFrame, use the following HTML:
```

... and then put the HTML there. But you also need to get rid of `http://127.0.0.1:5000`. In the docs, very specifically reference `http://crwa-flagging.herokuapp.com/`.